### PR TITLE
(maint) Allow the rpm spec scriptlets to be run with bash, not sh

### PR DESCRIPTION
**Pull request by melissa**
There are some issues, like creating the homedir for a new user, that sh
fails out when trying to do. Bash is a more stable way to run these
scriptles.

### CODE REVIEW

**Comment by melissa** [comment]
@haus I think I'm doing something wrong with this... thoughts?

File: templates/rpm/project.spec.erb [hunk 1]
```diff
@@ -114,7 +114,7 @@ for entry in `cat %{SOURCE1}`; do
   fi
 done
 
-%pre
+%pre -p <%= @bash %>
```
Comment (haus) [inline templates/rpm/project.spec.erb]: `@platform.bash`

File: lib/vanagon/platform/rpm.rb [hunk 2]
```diff
@@ -69,6 +69,7 @@ def initialize(name)
           @install = "/opt/freeware/bin/install"
         end
         @rpmbuild = "/usr/bin/rpmbuild"
+        @bash = "/bin/bash"
```
Comment (haus) [inline lib/vanagon/platform/rpm.rb]: Where does bash live on various platforms?

Comment (stahnma) [inline lib/vanagon/platform/rpm.rb]: Not in /bin on AIX.

Comment (melissa) [inline lib/vanagon/platform/rpm.rb]: I want to wait until Haus' other pull request refactoring this is merged, and then we'll have separate definitions for aix, rpm, etc.

**Comment by melissa** [comment]
This may not actually be needed...

**Pull request closed without merging**